src/styles/global.css

```
@@ -82,11 +82,13 @@ h1, h2, h3, h4, h5 {
   line-height: 1.3;
 }
 
-h1 {
+/* Site brand wordmark in the header (not a heading — see Header.astro). */
+.brand-name {
+  margin: 0;
   font-family: 'Orbitron', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)